github-activity-tracker/.env.example

```
@@ -0,0 +1,18 @@
+# GitHub Personal Access Token (required for sync endpoints)
+# Create at: https://github.com/settings/tokens (scope: repo)
+GITHUB_TOKEN=ghp_your_token_here
+
+# Database - use RDS_* for external DB (e.g. AWS RDS); or POSTGRES_* for Docker db
+RDS_HOST=your-rds-host-or-db
+RDS_PORT=5432
+RDS_DATABASE=github_tracker
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RDS_DATABASE key should go before the RDS_HOST key

(UnorderedKey)